plugins/woocommerce/templates/checkout/payment.php

```diff
@@ -31,7 +31,7 @@
 				}
 			} else {
 				echo '<li>';
-				wc_print_notice( apply_filters( 'woocommerce_no_available_payment_methods_message', WC()->customer->get_billing_country() ? esc_html__( 'Sorry, it seems that there are no available payment methods for your state. Please contact us if you require assistance or wish to make alternate arrangements.', 'woocommerce' ) : esc_html__( 'Please fill in your details above to see available payment methods.', 'woocommerce' ) ), 'notice' ); // phpcs:ignore WooCommerce.Commenting.CommentHooks.MissingHookComment
+				wc_print_notice( apply_filters( 'woocommerce_no_available_payment_methods_message', WC()->customer->get_billing_country() ? esc_html__( 'Sorry, it seems that there are no available payment methods. Please contact us if you require assistance or wish to make alternate arrangements.', 'woocommerce' ) : esc_html__( 'Please fill in your details above to see available payment methods.', 'woocommerce' ) ), 'notice' ); // phpcs:ignore WooCommerce.Commenting.CommentHooks.MissingHookComment
```

Comment:
Given the check for the billing country (`WC()->customer->get_billing_country()`), it seems like "state" here is actually in a geographic context rather than a system context. But I agree that "state" can be confusing since it might be interpreted specifically as a US state rather than an arbitrary sovereign entity. What about changing it to "location"?

> Sorry, it seems that there are no available payment methods for your location. Please contact us if you require assistance or wish to make alternate arrangements.

Reply:
That could also work! My only concern is that `for your location` implies that another geographic setting would resolve the issue—which isn't always the case.

Reply:
🤔 What other situations would cause there to not be an available payment method? The only one I can think of is if the merchant hasn't actually set up any payment methods at all...

Reply:
If a merchant is using [WooPayments](https://woocommerce.com/products/woopayments/?aff=10486&cid=1131038) without an additional payment gateway, but the account can no longer accept payments because it is restricted or rejected, then this notice will appear. 

While there are definitely other ways that this specific scenario is communicated to a merchant, this is what specifically made me want to submit this PR in the first place, as I am the product ambassador for WooPayments :)

This can also happen when other gateways are misconfigured (e.g., incorrect/missing API keys in [Stripe](https://woocommerce.com/products/stripe/?aff=10486&cid=1131038)) or the attached account is unable to accept payments.

Reply:
Gotcha, that makes sense. I think my concern was that "Sorry, it seems that there are no available payment methods" doesn't really provide a hint about the cause of the issue, but I guess if it's due to merchant error rather than geography, it's probably better not to have a hint about that on the front end anyway.